src/main/java/com/cupid/jikting/jwt/service/JwtService.java

```diff
@@ -76,23 +76,14 @@ public Optional<String> extractRefreshToken(HttpServletRequest request) {
                 .map(refreshToken -> refreshToken.replace(BEARER, REMOVE));
     }
 
-    public Optional<String> extractAccessToken(HttpServletRequest request) {
-        return Optional.ofNullable(request.getHeader(accessHeader))
-                .filter(refreshToken -> refreshToken.startsWith(BEARER))
-                .map(refreshToken -> refreshToken.replace(BEARER, REMOVE));
-    }
-
-    public Optional<Long> extractMemberProfileId(String accessToken) {
-        try {
-            return Optional.ofNullable(JWT.require(Algorithm.HMAC512(secretKey))
-                    .build()
-                    .verify(accessToken)
-                    .getClaim(MEMBER_PROFILE_ID_CLAIM)
-                    .asLong());
-        } catch (Exception e) {
-            log.error("액세스 토큰이 유효하지 않습니다.");
-            return Optional.empty();
-        }
+    public Long extractMemberProfileId(HttpServletRequest request) {
+        String accessToken = extractAccessToken(request);
+        return Optional.ofNullable(JWT.require(Algorithm.HMAC512(secretKey))
+                        .build()
+                        .verify(accessToken)
```

Comment:
p2. accessToken은 바로 사용해도 될 것 같아요!

```suggestion
        return Optional.ofNullable(JWT.require(Algorithm.HMAC512(secretKey))
                        .build()
                        .verify(extractAccessToken(request))
```